src/main/java/redis/clients/jedis/MultiNodePipelineBase.java

```diff
@@ -136,6 +137,15 @@ public final void sync() {
       log.error("Thread is interrupted during sync.", e);
     }
 
+    executorService.shutdown();
+    try {
+      if (!executorService.awaitTermination(5, TimeUnit.MINUTES)) {
+        executorService.shutdownNow();
+      }
+    } catch (InterruptedException e) {
+      executorService.shutdownNow();
+    }
+
```

Comment:
```suggestion
    try {
      executorService.shutdown();
    } catch (Exception e) {
      log.error("Error when shutdown executorService", e);
    }
```
If we get here, then there must be no tasks in ExecutorService, just shutdown?

Reply:
I agree, I think there is nothing wrong with using `executorService.shutdownNow()` directly here.